grafana-plugin/src/containers/RemoteSelect/RemoteSelect.tsx

```diff
@@ -127,7 +124,7 @@ const RemoteSelect = inject('store')(
         isSearchable={showSearch}
         value={value}
         onChange={onChangeCallback}
-        defaultOptions={options}
+        defaultOptions
```

Comment:
what is this change for?

Reply:
from [specs](https://developers.grafana.com/ui/latest/index.html?path=/docs/forms-select--auto-menu-placement)

```
defaultOptions | When specified as boolean the loadOptions will execute when component is mounted
```
it exactly what we need

Reply:
gotcha, thanks for clarifying 👍